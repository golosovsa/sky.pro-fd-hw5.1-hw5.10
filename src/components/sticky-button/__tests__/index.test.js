import ShallowRenderer from 'react-test-renderer/shallow'
import StickyButton from '..'

describe('<StickyButton />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<StickyButton>children</StickyButton>)

      expect(tree).toMatchSnapshot()
    })
  })
})
