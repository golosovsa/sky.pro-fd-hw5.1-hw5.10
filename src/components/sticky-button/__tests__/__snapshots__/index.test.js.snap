// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StickyButton /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.div
  active={false}
  onClick={[Function]}
>
  children
</styled.div>
`;
